libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/ImageList.tsx

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  Text,
+  FocusZone,
+  List,
+  Image,
+  ImageFit,
+  IRectangle
+} from "@fluentui/react";
+import React from "react";
+
+import { imageListStyles } from "./ImageList.styles";
+
+export interface IImageListProps {
+  imageDim: number;
+  openPanel(): void | undefined;
```

Comment:
did you mean void => void as the function definition?
```
openPanel: () => void;
```
If it is optional then:
```
openPanel?: () => void;
```
I'm just looking at other definitions in the codebase like:
https://github.com/microsoft/responsible-ai-toolbox/blob/4e97f7f20869d4532144c9a0cdb7f2de8fd53002/libs/error-analysis/src/lib/ErrorAnalysisDashboard/Interfaces/IErrorAnalysisDashboardProps.ts#L44